DeusEx/Classes/DXRFlags.uc

```diff
@@ -1,83 +1,107 @@
 class DXRFlags extends DXRBase;
 
-var FlagBase flags;
+var transient FlagBase f;
 
 //rando flags
 var int seed;
 var int flagsversion;//if you load an old game with a newer version of the randomizer, we'll need to set defaults for new flags
 var int brightness, minskill, maxskill, ammo, multitools, lockpicks, biocells, medkits, speedlevel;
-var int keysrando;//0=off, 1=dumb, 2=on (old smart), 3=copies
+var int keysrando;//0=off, 1=dumb, 2=on (old smart), 3=copies, 4=smart (v1.3), 5=path finding?
 var int doorspickable, doorsdestructible, deviceshackable, passwordsrandomized, gibsdropkeys;//could be bools, but int is more flexible, especially so I don't have to change the flag type
 var int autosave;//0=off, 1=first time entering level, 2=every loading screen
-var int removeinvisiblewalls, enemiesrandomized;
+var int removeinvisiblewalls, enemiesrandomized, infodevices;
+
+function PreTravel()
+{
+    Super.PreTravel();
+    f = None;
+    Self.Destroy();// for some reason, "f = tdxr.Player.FlagBase;" inside the Init function crashes if I don't do this, not sure why
+}
 
 function Init(DXRando tdxr)
 {
     Super.Init(tdxr);
-
-    flags = dxr.Player.FlagBase;
+    f = tdxr.Player.FlagBase;
+    SetTimer(1.0, True);
```

Comment:
maybe I should LoadFlags or InitDefaults before SetTimer? the timer could run SaveFlags